### PR TITLE
[14.0][ENH] Add Notification message on restart tier

### PR DESCRIPTION
**Pull request by newtratip**
I use sudo to post message follow this PR https://github.com/OCA/server-ux/pull/359

Pls review @kittiu

### CODE REVIEW

**Comment by kittiu** [comment]
Test and Code Review OK,

I understand that you want to add a message on click "Restart Validation" button, as currently there is no log in chatter.

If anything was confused is the the wording `Restart Validation` which is not really  "Restarting", but only "Reset" (stop/remove tiers), and user will need to click "Request Validation" again. (but I can't think of any other suitable name)

I think the log message should be changed to "The review has been reset by Mr.A".

image

File: base_tier_validation/models/tier_validation.py [hunk 1]
```diff
@@ -388,11 +391,23 @@ def request_validation(self):
         self._notify_review_requested(created_trs)
         return created_trs
 
+    def _notify_restarted_review_body(self):
+        return _("A review was restarted by %s.") % (self.env.user.name)
```
Comment (kittiu) [inline base_tier_validation/models/tier_validation.py]:
```suggestion
        return _("The review has been reset by %s.") % (self.env.user.name)
```

**Review by kittiu — APPROVED** [review]

File: base_tier_validation/models/tier_validation.py (hunk 1 above)
Comment (newtratip) [reply base_tier_validation/models/tier_validation.py]: Fixed.

**Review by Saran440 — APPROVED** [review]
LGTM :+1:

**Review by CasVissers-360ERP — APPROVED** [review]
👍

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**